test/SmartDetectorsExtensionsTests/SmartDetectorsExtensionsTests.csproj

```diff
@@ -16,6 +16,7 @@
     <PackageReference Include="StyleCop.Analyzers" Version="1.0.2" />
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\..\src\SDK\MonitoringApplianceEmulator\MonitoringApplianceEmulator.csproj" />
```

Comment:
>MonitoringApplianceEmulator [](start = 73, length = 27)

why?

Reply:
removed

---
In reply to: [257937497](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/48#discussion_r257937497) [](ancestors = 257937497)